akka-remote/src/main/scala/akka/remote/serialization/DaemonMsgCreateSerializer.scala

```diff
@@ -116,10 +99,28 @@ private[akka] class DaemonMsgCreateSerializer(val system: ExtendedActorSystem) e
 
     def props = {
       import scala.collection.JavaConverters._
-      val clazz = system.dynamicAccess.getClassFor[AnyRef](proto.getProps.getClazz).get
-      val args: Vector[AnyRef] = (proto.getProps.getArgsList.asScala zip proto.getProps.getClassesList.asScala)
-        .map(deserialize)(collection.breakOut)
-      Props(deploy(proto.getProps.getDeploy), clazz, args)
+      val protoProps = proto.getProps
+      val actorClass = system.dynamicAccess.getClassFor[AnyRef](protoProps.getClazz).get
+      val args: Vector[AnyRef] =
+        if (protoProps.getSerializerIdsCount > 0) {
```

Comment:
I don't think it is worth it, but I can clarify by moving the comment from the row below and be more explicit about the serializer ids not being present in old wire messages